.azure/pipelines/pr/Phonebook.pr.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
       - task: HelmInstaller@1
         inputs:
-          helmVersionToInstall: 'latest'
+          helmVersionToInstall: '2.14.2'
```

Comment:
This is not really sustained, right?

Reply:
What do you mean? Did you mean if the change is on purpose? Then the answer is yes.

Reply:
I mean that not the latest helm version will installed and it need to be updated in the future with a new pr. But this is probably not a big problem and have not a big impact.

Reply:
It has, that's why I am freezing the version number. The new helm release (v3) has some breaking changes. We should update in the future but because this pr is not about updating helm I just made the pipeline go green with this change.